package.json

```diff
@@ -734,6 +734,14 @@
                     "command": "gatewaysExplorer.testSmartContractEntry",
                     "when": "view == gatewaysExplorer && viewItem == blockchain-instantiated-multi-contract-item || viewItem == blockchain-instantiated-contract-item || viewItem == blockchain-contract-item || viewItem == blockchain-instantiated-unknown-item"
                 },
+                {
+                    "command": "reactPage.open",
```

Comment:
Might be worth change `reactPage.open` to `transactionPage.open` or something similar.